data/ecosystems/h/hedera.toml

```diff
@@ -1,98 +1,31 @@
 # Ecosystem Level Information
 title = "Hedera"
 
-sub_ecosystems = ["Calaxy"]
-
-github_organizations = ["https://github.com/hashgraph"]
+sub_ecosystems = [
+"Calaxy", 
+"Heliswap", 
+"Hashport",
+"SaucerSwap Labs",
+"Heliswap",
+"Buidler Labs",
+"Blade Labs",
+"Burstall",
+"PieFi Platform",
+"Mintmaster Labs",
+"HashGuild",
+"TxMQ, Inc",
+"HSuite",
+"Trust Enterprises",
+"DOVU",
+]
+
+github_organizations = [
+"https://github.com/hashgraph",
+"https://github.com/limechain",
```

Comment:
Can you remove this since limechain does some other repos non-specific to hedera.

Reply:
Whoops — yes, will update; thank you for flagging and apologies.